### PR TITLE
Fix the uploadimage form

### PR DESCRIPTION
**Pull request by hcyuser**

### CODE REVIEW

**Comment by jackycute** [comment]
Hi @hcyuser,

Thanks for contribution.
Can you explain what is this PR solving?

**Comment by hcyuser** [comment]
When I set up the image upload in local file system, I found I need to change _path_ key into _filepath_, which means the structure of _files_ changed.

**Comment by jackycute** [comment]
Oops, sorry clicking wrong button.

**Comment by jackycute** [comment]
@hcyuser Can you describe the error message like paste the error text or screenshot?
I think this is caused by https://github.com/node-formidable/formidable/pull/689

File: lib/imageRouter/index.js [hunk 1]
```diff
@@ -62,4 +62,4 @@ imageRouter.post('/uploadimage', function (req, res) {
       })
     }
   })
-})
+})
```
Comment (jackycute) [inline lib/imageRouter/index.js:65]: you need to leave a newline here to pass the standard lint.

**Comment by hcyuser** [comment]
> @hcyuser Can you describe the error message like paste the error text or screenshot?
> 
> I think this is caused by https://github.com/hackmdio/codimd/pull/1789 and was changed from https://github.com/node-formidable/formidable/pull/689

I got _Image path is missing or wrong_ on the terminal. After that, I found the structure of **files** changed, and therefore I modified it.

**Comment by Yukaii** [comment]
closed in favor of #1836

**Pull request closed without merging**